src/main/java/com/fleotadezuta/youthprogrammanager/facade/TicketChildTicketTypeFacade.java

```diff
@@ -29,12 +31,24 @@ public Flux<TicketTypeDto> getPotentialTicketTypes(String name) {
         return ticketTypeService.findByName(name);
     }
 
-    public Flux<TicketDto> getAllTickets() {
-        return ticketService.findAll()
-                .map(ticketMapper::fromTicketDtoToTicketDocument)
-                .flatMap(ticketDoc ->
-                        getTicketDtoWithChildAndTicketType(ticketDoc, ticketDoc.getChildId(), ticketDoc.getTicketTypeId())
-                );
+    public Flux<TicketDto> getAllTickets(UserDetails userDetails) {
+        if (userDetails.getUserType().equals("ADMIN")) {
```

Comment:
Ezek a `UserType`-ok kerüljenek ki egy enum-ba és hivatkozz arra mindenhol, ahol beállításra kerül vagy ellenőrizve van.

Korábban is volt ilyen valószínűleg, nézd majd át mindenhol.